app/core/src/main/java/com/fsck/k9/Account.java

```diff
@@ -821,6 +823,15 @@ public String getDisplayName() {
         return description != null ? description : getEmail();
     }
 
+    public Drawable getIcon() {
```

Comment:
I like this idea, but please move this into a separate class. Account is overloaded as it is.

Reply:
Well, it somehow belongs there, especially if we want to reuse the icons when AccountsLiveData is ready. However, if we switch to materialdrawer.AccountHeader it would make sense to use ContactPictures here, I guess. I'll have a look...